qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -161,10 +163,16 @@ def solve(
         matrix_operators_dict, size = self._prepare_matrix_operators(problem)
 
         # 3. Evaluate eom operators
-        measurement_results = self._gsc.evaluate_operators(
-            groundstate_result.eigenstates[0], matrix_operators_dict
+        estimator = getattr(self._gsc.solver, "estimator", None)
+        if estimator is None:
+            estimator = Estimator()
```

Comment:
This is not ideal right now. We had also discussed whether providing the user the ability to provide an `estimator` via the `QEOM.__init__` would be better.
In fact, should we require that to be done? The user can always provide the same primitive in multiple places :+1: